guides-and-tutorials/running-a-signer/signer-quickstart.md

```diff
@@ -0,0 +1,548 @@
+db_path = "$HOME/stacks-signer/data/signer.sqlite"
+auth_password = "$AUTH_TOKEN"
+stacks_private_key = "$PRIVATE_KEY"
+metrics_endpoint = "127.0.0.1:9154"
+block_proposal_timeout_ms = 180000
+EOF
+```
+
+{% endtab %}
+
+{% endtabs %}
+
+### Verify the setup
+
+To ensure the signer has been set up correctly, you can run the following commands:
+
+{% tabs %}
+
+{% tab title="Mainnet" %}
+
+```bash
+# Verify the signer's version
+~/stacks-signer/stacks-signer --version
+
+# Output:
+stacks-signer signer-2.5.0.0.5.2 (release/signer-2.5.0.0.5.2:d441e02, release build, linux [x86_64])
+
+# Verify the config file
+~/stacks-signer/stacks-signer check-config -c ~/stacks-signer/signer-config.toml 
+
+# Output:
+Config: 
+Stacks node host: 127.0.0.1:20443
+Signer endpoint: 127.0.0.1:30000
+Stacks address: SP1G... # address from keychain file
+Public key: 03a3... # publicKey from keychain file
+Network: mainnet
+Database path: ~/stacks-signer/data/signer.sqlite
+DKG transaction fee: 0.01 uSTX
+Metrics endpoint: 127.0.0.1:9154
+```
+
+{% endtab %}
+
+{% tab title="Testnet" %}
+
+```bash
+# Verify the signer's version
+~/stacks-signer/stacks-signer --version
+
+# Output:
+stacks-signer 3.0.0.0.0-rc5 (release/3.0.0.0.0-rc5:3e4ae4f, release build, linux [x86_64])
+
+# Verify the config file
+~/stacks-signer/stacks-signer check-config -c ~/stacks-signer/signer-config.toml 
+
+# Output:
+Config: 
+Stacks node host: 127.0.0.1:20443
+Signer endpoint: 127.0.0.1:30000
+Stacks address: SP1G... # address from keychain file
+Public key: 03a3... # publicKey from keychain file
+Network: testnet
+Chain ID: 0x80000000
+Database path: ~/stacks-signer/data/signer.sqlite
+Metrics endpoint: 127.0.0.1:9154
+```
+
+{% endtab %}
+
+{% endtabs %}
+
+### Start the signer
+
+If the outputs of the previous commands are correct, you can proceed and start the signer:
+
+```bash
+~/stacks-signer/stacks-signer run -c ~/stacks-signer/signer-config.toml
+```
+
+## Step 3 - Set Up Your Stacks Node
+
+### Download the stacks-node binary
+
+Official binaries are available from the [Stacks Core releases page on Github](https://github.com/stacks-network/stacks-core/releases). Each release includes pre-built binaries. Download the [latest node release ZIP file](https://github.com/stacks-network/stacks-core/releases/latest) for your server’s architecture and decompress it. Inside of that folder is a `stacks-node` binary. Currently, the latest mainnet and testnet versions that you have to use differ.
+
+Assuming a `Linux x64` machine, the commands to download and uncompress the node binary look like this:
+
+{% tabs %}
+
+{% tab title="Mainnet" %}
+
+```bash
+# Enter the node directory
+cd ~/stacks-node
+
+# Download the node binary zip
+wget https://github.com/stacks-network/stacks-core/releases/download/2.5.0.0.7/linux-glibc-x64.zip
+
+# Unzip the node binary archive
+unzip linux-glibc-x64.zip
+
+# Remove unwanted files
+find ~/stacks-node -type f ! -name 'stacks-node' -delete
```

Comment:
i wouldn't add this, since the only benefit is saving some disk space (marginal)